conda_smithy/tests/test_lint_recipe.py

```diff
@@ -388,6 +388,18 @@ def test_maintainer_exists(self):
         lints = linter.lintify({'package': {'name': 'python'}}, recipe_dir='recipe', conda_forge=False)
         self.assertNotIn(expected_message, lints)
 
+        expected_message = 'Recipe with the same name exists in bioconda'
+        # Check that feedstock exists if staged_recipes
+        lints = linter.lintify({'package': {'name': 'abawaca'}}, recipe_dir='abawaca', conda_forge=True)
+        self.assertIn(expected_message, lints)
+        lints = linter.lintify({'package': {'name': 'abawaca'}}, recipe_dir='abawaca', conda_forge=False)
+        self.assertNotIn(expected_message, lints)
+        # No lint if in a feedstock
+        lints = linter.lintify({'package': {'name': 'abawaca'}}, recipe_dir='recipe', conda_forge=True)
+        self.assertNotIn(expected_message, lints)
+        lints = linter.lintify({'package': {'name': 'abawaca'}}, recipe_dir='recipe', conda_forge=False)
+        self.assertNotIn(expected_message, lints)
```

Comment:
Can you move this block below line 416 and also add a test like in Line 404-416?

Reply:
Yep, looked at this block too quickly last night; fixing.